lib/Command/LintCommand.php

```diff
@@ -22,7 +22,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $processes = [];
         $processes[] = $this->asyncProc(['find', $dir, '-name', '*.yml', '!', '-path', '*/vendor/*', '-exec', 'vendor/bin/yaml-lint', '{}', ';']);
-        $processes[] = $this->asyncProc(['find', $dir, '-name', '*.php', '!', '-path', '*/vendor/*', '-exec', 'php', '-l', '{}', '2>&1', ';']);
+        $processes[] = $this->asyncProc(['vendor/bin/parallel-lint', '--exclude',  'vendor', $dir]);
```

Comment:
Vermutlich ist dieses neue exclude nur für /vendor, aber nicht wie zuvor für /vendor/ in beliebigen unterverzeichnissen


Todo: testen

Reply:
getestet... macht genau das gleiche wie das alte command.